bindings/tests/Capgemini.PowerApps.SpecFlowBindings.UiTests/Hooks/MockSolutionHooks.cs

```diff
@@ -24,9 +27,11 @@ public static void ImportMockSolution()
         {
             using (var serviceClient = GetServiceClient())
             {
-                serviceClient.ImportSolutionToCrm(SolutionPath, out var importId);
-
-                if (serviceClient.LastCrmException != null)
+                if (serviceClient != null && serviceClient.IsReady)
+                {
+                    InstallSolution(serviceClient, SolutionPath, TimeSpan.FromMinutes(5), TimeSpan.FromSeconds(30));
+                }
+                else
                 {
                     throw serviceClient.LastCrmException;
                 }
```

Comment:
Inverting the condition might reduce nesting:

```suggestion
                if (serviceClient == null || !serviceClient.IsReady)
                {
                    throw serviceClient.LastCrmException;
                }
                
                InstallSolution(serviceClient, SolutionPath, TimeSpan.FromMinutes(5), TimeSpan.FromSeconds(30));
```